### ISSUE TITLE
Taskbar Volume Control - does not change by "step" size in Win10 Indicator

### ISSUE THREAD

**Issue by ttodua** [ramensoftware/windhawk-mods#939]
When "Volume indicator" style is set to `Win 11` , the mousewheel scroll change value by "step" size setting (as expected) however, when selected `Win 10`, volume indicator only changes by  `2` step-size on mousewheel scroll.

**Comment by ttodua** [comment]
@m417z

**Comment by ttodua** [comment]
also here:
https://github.com/ramensoftware/windhawk-mods/blob/c17ce99483d75133beaed805c705e10872660059/mods/taskbar-volume-control.wh.cpp#L802
I think that there should be `nStep = g_settings.volumeChangeStep;` in the beginning, it shouldn't be done inside `if (!nStep)` clause. I undersatnd it might not be consistent (eg. if you have setting of `15` as step-size, then first jump will be `15` but next scrolls would be still 2).
however, that's better than permanent 2, because **at least** the first scroll would respect our settings

**Comment by m417z** [comment]
Sorry for the delay. I finally found the time to look at it. Please try this version:
https://github.com/m417z/my-windhawk-mods/blob/0a9a4216f69216d7b7c1c32656a257101a4164ee/mods/taskbar-volume-control.wh.cpp

**Comment by ttodua** [comment]
> Fixed in version 1.3.

thanks

**Comment by m417z** [comment]
Right, the custom step is only used when the mouse is over the taskbar. Adding support for scrolling over the volume indicator (which is a separate process) requires either a global mouse hook, which may affect performance, or customizing these processes as well, which requires additional work. I don't think it's worth it, since I believe you usually just scroll over the taskbar.
